environment/src/main/java/com/sequenceiq/environment/experience/api/CommonExperienceApi.java

```diff
@@ -0,0 +1,15 @@
+package com.sequenceiq.environment.experience.api;
+
+import java.util.Set;
+
+import javax.validation.constraints.NotNull;
+
+import com.sequenceiq.environment.experience.common.responses.DeleteCommonExperienceWorkspaceResponse;
+
+public interface CommonExperienceApi {
+
+    @NotNull Set<String> getWorkspaceNamesConnectedToEnv(String experienceBasePath, String environmentCrn);
```

Comment:
why would we rename it? I mean I'm not saying that we must have to keep this word but if you can suggest a better one that is related – and can be "connected" easily – to this remote resource (or set of resources to be precise), I'm open to change it 🙂

Reply:
unfortunately, I can't. I tried to figure out why workspace, but. I couldn't find it. I suggest the name changing, because we have a reserved word for the workspace and I think it will cause some problem when trying to understand the meaning here.

Of course, this is a suggestion.